src/pages/ProductDetail/Count.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './Count.scss'
+
+const Count = ({ id, quantity, setQuantity }) => {
```

Comment:
```suggestion
const Count = ({ quantity, setQuantity }) => {
```
사용하지 않는 부분이라면 지워도 좋습니다.